presto-main/src/main/java/com/facebook/presto/sql/planner/PartitioningScheme.java

```diff
@@ -29,10 +29,40 @@
 
 public class PartitioningScheme
 {
+    public enum Replication
+    {
+        NONE {
+            @Override
+            public boolean isNoReplication()
```

Comment:
I had `replication == NONE` but then extracted the method. If some day i change the enum to some configurable being, I will surely need to revisit all usage places but I will not have to fix them all.